js/app/vite.config.ts

```diff
@@ -124,7 +125,8 @@ export default defineConfig(({ mode }) => {
 						}
 					}
 				}
-			}
+			},
+			Sonda()
```

Comment:
This is cool but do we want the Sonda page to open up every time we build the frontend? Perhaps we could make this only open up if a cmd line argument is provided